### PR TITLE
update balancing algorithm

### LINKED ISSUE #821: Increase speed of Quota balancing
1. to ensure max balancing speed, set "cfs.quotaCacheMoveMax DN 50", set "cfs.quotaCacheDivisorFloor" to smallest percent, set "cfs.maxMultiplierForInvalidatedMassQuota VS 2"

1. For each core filer: 
    1. Create core filer
    1. set the policy "cpolicyActiveMASSID.fixedQuota RU PERCENT_SPACE" where MASSID is the id of the mass, and PERCENT_SPACE is the desired present space

1. Wait 30 minutes or until desired balance is achieved (by checking "analytics.getCoreFilerCacheSpaceData")

1. to enable dynamic change, unset all cfs.quotaCacheMoveMax, unset all quotaCacheDivisorFloor, unset maxMultiplierForInvalidatedMassQuota

### PR DESCRIPTION
**Pull request by anhowe**
fixes #821

### CODE REVIEW

File: src/terraform/providers/terraform-provider-avere/resource_vfxt.go [hunk 1]
```diff
@@ -1143,16 +1150,24 @@ func createCoreFilers(coreFilersToAdd map[string]*CoreFiler, averevfxt *AvereVfx
...
 
 func startFixedQuotaPercent(corefilers []*CoreFiler, averevfxt *AvereVfxt) error {
-	if isRequired := isFixedQuotaRequired(corefilers); isRequired {
+	if smallestQuota, isRequired := isFixedQuotaRequired(corefilers); isRequired {
 		if err := averevfxt.CreateCustomSetting(QuotaCacheMoveMax); err != nil {
 			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
 		}
+
+		if err := averevfxt.CreateCustomSetting(fmt.Sprintf(QuotaDivisorFloor, smallestQuota)); err != nil {
+			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
```
Comment (baileye) [inline src/terraform/providers/terraform-provider-avere/resource_vfxt.go:1159]: The error should print the quotaDivisorFloor information

File: src/terraform/providers/terraform-provider-avere/resource_vfxt.go [hunk 2]
```diff
@@ -1143,16 +1150,24 @@ func createCoreFilers(coreFilersToAdd map[string]*CoreFiler, averevfxt *AvereVfx
...
+		}
+
+		if err := averevfxt.CreateCustomSetting(QuotaMaxMultiplierForInvalidatedMassQuota); err != nil {
+			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
```
Comment (baileye) [inline src/terraform/providers/terraform-provider-avere/resource_vfxt.go:1163]: Error should print QuotaMaxMultiplierForInvalidatedMassQuota

File: src/terraform/providers/terraform-provider-avere/resource_vfxt.go [hunk 3]
```diff
@@ -1174,17 +1189,24 @@ func finishFixedQuotaPercent(corefilers []*CoreFiler, averevfxt *AvereVfxt) erro
 		if err := averevfxt.RemoveCustomSetting(QuotaCacheMoveMax); err != nil {
 			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
 		}
+		if err := averevfxt.RemoveCustomSetting(QuotaDivisorFloor); err != nil {
+			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
```
Comment (baileye) [inline src/terraform/providers/terraform-provider-avere/resource_vfxt.go:1193]: Error should print QuotaDivisorFloor

File: src/terraform/providers/terraform-provider-avere/resource_vfxt.go [hunk 4]
```diff
@@ -1174,17 +1189,24 @@ func finishFixedQuotaPercent(corefilers []*CoreFiler, averevfxt *AvereVfxt) erro
...
+		}
+		if err := averevfxt.RemoveCustomSetting(QuotaMaxMultiplierForInvalidatedMassQuota); err != nil {
+			return fmt.Errorf("ERROR: failed to apply custom setting '%s': %s", QuotaCacheMoveMax, err)
```
Comment (baileye) [inline src/terraform/providers/terraform-provider-avere/resource_vfxt.go:1196]: Error should print QuotaMaxMultiplierForInvalidatedMassQuota

File: src/terraform/providers/terraform-provider-avere/resource_vfxt.go (hunk 1 above)
Comment (anhowe) [reply src/terraform/providers/terraform-provider-avere/resource_vfxt.go]: fixed

File: src/terraform/providers/terraform-provider-avere/resource_vfxt.go (hunk 2 above)
Comment (anhowe) [reply src/terraform/providers/terraform-provider-avere/resource_vfxt.go]: fixed

File: src/terraform/providers/terraform-provider-avere/resource_vfxt.go (hunk 4 above)
Comment (anhowe) [reply src/terraform/providers/terraform-provider-avere/resource_vfxt.go]: fixed

File: src/terraform/providers/terraform-provider-avere/resource_vfxt.go (hunk 3 above)
Comment (anhowe) [reply src/terraform/providers/terraform-provider-avere/resource_vfxt.go]: fixed

**Review by baileye — APPROVED** [review]

**Pull request merged**